documentation/3.3/content/userguide/aks/_index.md

```diff
@@ -80,23 +78,24 @@ In this section, you can configure some options about the AKS which will run Web
+|Create a new Azure Container Registry to store application images?|If set to **Yes**, the offer will create a new Azure Container Registry (ACR) to hold the Docker images for use in the deployment.  If set to **No**, you must specify an existing ACR. In this case, you must be sure the selected ACR has the admin account enabled. For details, please see [Admin account](https://docs.microsoft.com/azure/container-registry/container-registry-authentication?tabs=azure-cli#admin-account). |
 | Select existing ACR instance | This option is shown only if **Use a pre-existing WebLogic Server Docker image from Oracle Container Registry?** is set to **No**. If visible, select an existing Acure Container Registry instance. |
 | Please provide the image path | This option is shown only if **Use a pre-existing WebLogic Server Docker image from Oracle Container Registry?** is set to **No**. If visible, the value must be a fully qualified Docker tag of an image within the specified ACR. |
 | Username for Oracle Single Sign-On authentication | The Oracle Single Sign-on account user name for which the Terms and Restrictions for the selected WebLogic Server image have been accepted. |
-| Password for Oracle Single Sign-On authentication | The password for that account. | 
-| Confirm password | Re-enter the value of the preceding field. |
+| Password for Oracle Single Sign-On authentication | The password for that account. |
+| Confirm password | Re-enter the value of the preceding field. | If 'Yes' is selected, the deployment process will pull from the CPU WebLogic Server image repository in the OCR. If 'No' is selected the deployment process will pull from the WebLogic Server image repository in OCR. |
+| Is the specified SSO account associated with an active Oracle support contract? | If set to **Yes**, you must accept the license agreement in `middleware/weblogic_cpu` repository. If set to **No**, you must accept the license agreement in `middleware/weblogic`. Steps to accept the license agreement: log on to [Oracle Container Registry](https://container-registry.oracle.com/); navigate to repository `middleware/weblogic_cpu` and `middleware/weblogic`; accept license agreement. See this [document](https://aka.ms/wls-aks-ocr-doc) for more information. |
```

Comment:
license agreement in `middleware/weblogic_cpu`  -> license agreement in the `middleware/weblogic_cpu` 
log on to -> log in to the
navigate to repository -> navigate to the `middleware/weblogic_cpu` and `middleware/weblogic` repositories
accept license agreement -> accept the license agreement

Reply:
See https://github.com/oracle/weblogic-kubernetes-operator/pull/2746/commits/fc56f0c0ed55c34ff3e80d6232e8f5f03d84e508